CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Deprecations
 
+* The `git.push-branch-prefix` config has been removed in favor of
+  `git.push-bookmark-prefix`.
```

Comment:
We list new deprecations under "Deprecations" and removals of already deprecated features under "Breaking changes" (so this should move there)